addons/web/static/src/views/view_button/view_button.xml

```diff
@@ -14,8 +14,8 @@
         t-att-type="clickParams.type"
         t-att-data-tooltip-template="hasBigTooltip ? 'views.ViewButtonTooltip' : false"
         t-att-data-tooltip-info="hasBigTooltip ? tooltip : false"
-        t-att-data-tooltip="hasSmallToolTip ? props.title : false"
```

Comment:
why that change. If it is really necessary --> remove getter for hasSmallToolTip

Reply:
I don't think it was expected, maybe I had a conflict and I remove that line. 🙈